eq-author-api/docker-compose.yml

```
@@ -28,3 +35,7 @@ services:
       - POSTGRES_DB=postgres
     ports:
       - 5432:5432
+  dynamo:
+    image: amazon/dynamodb-local
+    ports:
+      - 8000
```

This port conflicts with `go-launch-a-survey`.

```suggestion
      - 8050:8000
```